.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: rust
+rust:
+  - nightly
```

Comment:
Does this mean this PR makes Notify only build on Nightly? We aim to track Stable, so it would have to wait.